templates/legal/data-privacy/online-purchase.md

```diff
@@ -24,35 +24,104 @@ When you purchase products from us online, we ask you for your name, address, co
+Your information will be used by our third party payment provider to verify your credit card details for your purchase and for Canonical or our third party supplier to process your order and to send you your goods. We will also send you a receipt via email and we (or the third party supplier) may use your telephone number to contact you regarding your delivery.
 
 We require this information in order to process your payment, deliver your goods and fulfil our contract with you.
 
 ## What do we do with your information?
 
-Your information is stored in our order processing system and may be processed outside of the European Economic Area. It is shared with the third parties set out below and as reasonably required for Canonical to process and store your data.
+Your information is stored in our database and may be processed outside of the European Economic Area. 
 
-Your credit card details are passed to a third-party payment processor who retains credit card details in the European Economic Area or where required, is certified to the EU-US Privacy Shield (which requires effective safeguards for your information). We do not retain your credit card information.
+Such countries do not have the same data protection laws as the United Kingdom and EEA. Where the European Commission has not given a formal decision that such countries provide an adequate level of data protection, any transfer of your personal information will be subject to approved contract terms designed to help safeguard your privacy rights and give you remedies in the unlikely event of misuse of your personal information. If you would like further information please contact us (see “Your right to complain” below).  It is shared with the third parties set out below and as reasonably required for Canonical to process and store your data.
+
+Your credit card details are passed to a third-party payment processor who retains credit card details.
 
 Our third-party payment processor is a company called Stripe Payments Europe, Limited, who we use to process and confirm your payments. We share with them your name, address and credit card information only for payment purposes. We do not retain credit card data but our payment service providers do. For your safety, we have in place agreement between us to protect, safeguard your information and prohibit any sharing for any reasons whatsoever.
 
 We also use a third-party supplier called Shopify Inc. to provision your order and we share with them your name, address and telephone number in order to complete the delivery.
 
+## Our legal basis to use your Personal information
+
+#### *Our legitimate business interests*
```

Comment:
These should be h3 without emphasis

Reply:
@steverydz the "Your consent" (line 52) is written the same as "Our legitimate business interests" (line 45) in the copy doc, so should I change it to h3 no emphasis also?